### ISSUE TITLE
[Feature request] GUI for creating command binds

### ISSUE THREAD

**Issue by VasilisThePikachu** [space-wizards/space-station-14#36211]
For a while now, more advanced players have manually edited their ``keybinds.yml`` file in order to have emotes at a whim's notice

As an example
```
version: 1
binds:
# This bind will do the action "snaps their fingers" when you press the Home button on your keyboard
- key: Home
  type: Command
  function: me snaps their finger
# This bind will do the action "blows a raspberry" when you press the Control and page up button on your keyboard
- mod1: Control
  key: PageUp
  type: Command
  function: me blows a raspberry
```

Obviously... it's already annoying players to have to manually edit a file to get such a basic feature to enhance their role-play.

Another issue is that pretty commonly, people will mess up their YML formatting, which will cause the game to fall back and kill their key binds. And then come for support, not realizing they borked up their key binds file.

The [``bind``](https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Shared/Input/Binding/CommandBind.cs) command exists to make this a little easier, but as far as I can tell. Its changes are temporary, usable only in the client's runtime and can only support single keys (no modifiers)

I want to see the following:
-  A UI in the key binds options menu to easily allow adding, removing and editing command keybinds
- Allow you to just press the key to automatically add any modifier keys.
- Extra brownie points if you add a drop-down for say/emote so don't the player does not need to expliticly state the command `me` or `say`.

This mockup took me like 30mins to create [please appreciate it](https://i.redd.it/uu3ig2az6c121.jpg)
![Image](https://github.com/user-attachments/assets/cfe76698-f0f5-408f-a027-5de21e8f7c8a)

This is a very shit mockup, but hopefully it gets the message across

**Comment by Fildrance** [comment]
Popups are bad UI design and should be avoided at all costs (i mean, when possible) sono to u on that.

**Comment by Fildrance** [comment]
https://github.com/space-wizards/RobustToolbox/pull/6142/files
and
https://github.com/space-wizards/space-station-14/pull/39664

are kinda ready, and functional

**Comment by VasilisThePikachu** [comment]
> Popups are bad UI design and should be avoided at all costs (i mean, when possible) sono to u on that.

I mean that was just a shitty mockup to show my concept I like the concept you showed me better

**Comment by Fildrance** [comment]
RT implementation that i made was rejected as half-measure and needs a redesign, so issue is still relevant. UI part (https://github.com/space-wizards/space-station-14/pull/39664) could be reused, i suppose.
